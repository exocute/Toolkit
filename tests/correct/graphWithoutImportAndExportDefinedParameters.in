	// different names, same activity call

Graph graph
	Activity act1 act.ex
	Activity act2 act.ex
	connection [act1;act2]
