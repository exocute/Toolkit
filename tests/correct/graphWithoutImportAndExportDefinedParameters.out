Name: graph
Id: act1, Name: act.ex
Id: act2, Name: act.ex
act1:act2